tests/python/relay/test_op_qnn_conv2d.py

```diff
@@ -642,6 +642,11 @@ def test_broadcast_layout():
 
 
 def test_conv2d_int8():
+    target = "llvm -mcpu=core-avx2"
+    if not tvm.module.enabled(target):
+        print("skip because %s is not enabled..." % target)
+        return
+
```

Comment:
So, how do we run the same test for Arm or other CPU implementations ? 

I don't think the test infrastructure allows for this kind of execution yet but we probably need to start thinking about this and need helper functions across the test infrastructure that allows us to parameterize this on a per cpu target basis.

Ramana




Ramana

Reply:
Yes, we currently don't have a good way to check cpu features at runtime. The above is what we do in the unit test. I also think this is not a good way. 

The problem to used solved in this issue should be helpful?
https://github.com/dmlc/tvm/issues/4120

Reply:
Agreed -

 maybe we should create an issue to go clean all of this up once #4120 lands. which was why I wasn't requesting changes :)

Reply:
Exactly:)